src/main/java/com/codeus/winter/config/DependencyDescriptor.java

```diff
@@ -51,6 +58,10 @@ public Type getDependencyType() {
         return dependencyType;
     }
 
+    public Annotation[] getAnnotations() {
+        return annotations;
```

Comment:
Let's return a copy of the  `annotations` to avoid any unexpected updates from outside.

Reply:
Fixed